Add-ons/Umbraco-Deploy/Deploy-Settings/index.md

```diff
@@ -150,6 +151,14 @@ This setting defaults to 5 minutes.
 
 All of these times are configured using [standard timespan format strings](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-timespan-format-strings).
 
+## Batch settings
+
+Even with appropriate settings of the above timeouts, Deploy's backoffice transfer operations can hit a hard limit imposed by the Azure app service or other hosting environment. For Azure, this is around 4 minutes. This will normally only be reached if deploying a considerable amount of items in one go. For example, a media folder with thousands of items can reach this limit.
```

Comment:
```suggestion
Even with appropriate settings of the above timeouts, Deploy's backoffice transfer operations can hit a hard limit imposed by the hosting environment. For Azure, this is around 4 minutes. This will typically only be reached if deploying a considerable amount of items in one go. For example, a media folder with thousands of items can reach this limit.
```